src/gui/updownratiodlg.cpp

```diff
@@ -28,13 +28,16 @@
  * Contact : chris@qbittorrent.org
```

Comment:
@LordNyriox, there should be one name per line.
@naikel, leave it as is unless you want to add yourself.